setup.py

```diff
@@ -68,6 +68,7 @@ def run_tests(self):
         'yamale',
         'ruamel.yaml',
         'awsretry',
+        'lxml'
```

Comment:
We need lxml to parse the xml. If adding yet another kind of heavy dependency gives anybody heartburn, I could begrudgingly rewrite this to just use element tree parser in the standard library.